test/shared/index.js

```diff
@@ -18,7 +18,7 @@ exports.test = function(name) {
       var locals = { user: user };
       cons[name](path, locals, function(err, html){
         if (err) return done(err);
-        html.should.equal('<p>Tobi</p>');
+        html.should.match(/Tobi/);
```

Comment:
Does the `<p>...</p>` really has to be removed? I guess using match with the p should still be possible? And if the id is going to be kept, you might aswell add that as optional part in the regex to match that too.